test-data/unit/check-namedtuple.test

```diff
@@ -125,6 +128,8 @@ E = namedtuple('E', 'a b', 0)
 [builtins fixtures/bool.pyi]
 
 [out]
+main:4: error: Boolean literal expected as the "rename" argument to namedtuple()
+main:5: error: Boolean literal expected as the "rename" argument to namedtuple()
```

Comment:
A little unfortunate that we give two errors for the same line, but the same happens with `T = TypeVar("T", covariant="True")`; I don't think it's worth the effort to suppress one of the two.

Reply:
I think its fine to keep both errors but it would be better if both errors have the same code so that they can be disabled with a single type ignore comment. Currently the errors are:
```
nt.py:5: error: Boolean literal expected as the "rename" argument to namedtuple()  [misc]
nt.py:5: error: Argument "rename" to "namedtuple" has incompatible type "str"; expected "bool"  [arg-type]
```
How about giving the first error the code `arg-type` as well?

Reply:
Good idea, done.